src/common/utils/pagination.ts

```diff
@@ -0,0 +1,41 @@
+import { DeserializedList, PaginationOptions} from '../interfaces';
+// Autopaginatable class definition
+export class Autopaginatable<T> {
+  constructor(private _list: DeserializedList<T>, private apiCall: (params: PaginationOptions) => Promise<Autopaginatable<T>>) {}
```

Comment:
Your code editor might not be setup to run Prettier. (You can also do a `yarn format`).